edi_oca/tests/test_backend_jobs.py

```diff
@@ -31,6 +37,9 @@ def test_output(self):
         self.assertEqual(
             created.name, "Generate output content for given exchange record."
         )
+        # Check related jobs
+        self.assertEqual(len(created), record.related_queue_jobs_count)
```

Comment:
This is redundant as you compare the records below. Same for all the occurrences.

Reply:
Thank @simahawk , Is it better yet?